platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManagerTest.java

```diff
@@ -618,14 +618,275 @@ public void testDoImportRecordsMigrationExceptionWithInvalidChecksum() throws Ex
                 + ".dar]."));
   }
 
-  private void expectImportDelegationIsSuccessful()
-      throws NoSuchPaddingException, NoSuchAlgorithmException {
+  @Test
+  public void doDecryptSucceeds() throws Exception {
+    final Path path = ddfHome.resolve(TEST_DIRECTORY);
+
+    configurationMigrationManager = spy(getConfigurationMigrationManager());
+
+    // Need to actually generate a zip with a valid checksum so calling export
+    configurationMigrationManager.doExport(path);
+
+    expectImportDelegationIsSuccessful();
+
+    final MigrationReport report = configurationMigrationManager.doDecrypt(path);
+
+    assertThat("Decrypt was not successful", report.wasSuccessful(), is(true));
+    reportHasInfoMessage(
+        report.infos(),
+        equalTo(
+            "Successfully decrypted file ["
+                + path.resolve(TEST_BRANDING)
```

Comment:
❓ Think having private method(s) to build those `.dar` and `.zip` file names would make the code shorter/clearer?

Reply:
Refactored all existing tests as well.